index.bs

```diff
@@ -979,25 +979,35 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: This refers in general to the combination of the user's [=client platform=], [=authenticators=], and everything gluing
     it all together.
 
-: <dfn>Client-side-resident Public Key Credential Source</dfn>
-: <dfn>Resident Credential</dfn>
-:: A [=Client-side-resident Public Key Credential Source=], or [=Resident Credential=] for short,
-    is a [=public key credential source=] that is discoverable and usable in [=authentication ceremonies=]
-    without providing [=credential ID=]s,
-    i.e., calling {{CredentialsContainer/get()|navigator.credentials.get()}}
-    with an [=list/is empty|empty=] {{PublicKeyCredentialRequestOptions/allowCredentials}} argument.
-
-    As a consequence, a [=resident credential capable=] [=authenticator=] can generate an [=assertion signature=]
-    for a [=resident credential=] given only an [=RP ID=],
-    which in turn means that the [=public key credential source=]
+: <dfn>Client-side discoverable Public Key Credential Source</dfn>
+: <dfn>Client-side discoverable Credential</dfn>
+: <dfn>Discoverable Credential</dfn>
+: \[DEPRECATED] <dfn>Resident Credential</dfn>
+: \[DEPRECATED] <dfn>Resident Key</dfn>
+:: Note: Historically, [=client-side discoverable credentials=] have  been known as [=resident credentials=] or [=resident keys=].
+    Due to the phrases `ResidentKey` and `residentKey` being widely used in both the [=web authentication api|WebAuthn 
+    API=] and also in the [=Authenticator Model=] (e.g., in dictionary member names, algorithm variable names, and
+    operation parameters) the latter usage of `resident` within their
+    names has not been changed, i.e., for backwards compatibility purposes. Also, the term [=resident key=] is
+    defined here as equivalent to a [=client-side discoverable credential=].
```

Comment:
Seems to me like this "latter" isn't adding much value? Maybe the "i.e." is also redundant?

```suggestion
    Due to the phrases `ResidentKey` and `residentKey` being widely used in both the [=web authentication api|WebAuthn 
    API=] and also in the [=Authenticator Model=] (e.g., in dictionary member names, algorithm variable names, and
    operation parameters) the usage of `resident` within their
    names has not been changed for backwards compatibility purposes. Also, the term [=resident key=] is
    defined here as equivalent to a [=client-side discoverable credential=].
```